packages/design/src/Popover/Popover.jsx

```diff
@@ -203,6 +219,8 @@ export default class Popover extends React.Component {
     return {
       top: `${top}px`,
       left: `${left}px`,
+      bottom: `${containerWindow.innerHeight - bottom}px`,
+      right: `${containerWindow.innerWidth - right}px`,
```

Comment:
Before the initial definition of `bottom` and `right`, I'd also be for adding this comment, otherwise the variable names might be a bit misleading.

```typescript
    // bottom and right correspond to the calculated position of the element from the top left, not
    // from the bottom right, meaning they must be inverted before using them as `bottom` and
    // `right` CSS properties.
    let bottom = top + elemRect.height;
    let right = left + elemRect.width;
```

Reply:
Thanks, done!